ionic2/app/components/poke-details/poke-type.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+  template: require('./poke-type.component.html'),
+  selector: 'poke-type',
+  styles: [`
```

Comment:
I know it's not a lot of CSS but I think it would still be nice to move it to a separate file rather than inlining it.